.github/workflows/coderabbit-autofix.yml

```
@@ -0,0 +1,469 @@
+          if [[ "$FIRST_PASS" == "true" || "$RETRY_PASS" == "true" ]]; then
+            echo "final_passed=true" >> $GITHUB_OUTPUT
+          else
+            echo "final_passed=false" >> $GITHUB_OUTPUT
+          fi
+
+      # COMMIT & PUSH: Only if validation passed
+      - name: Commit and push fixes
+        if: steps.final-status.outputs.final_passed == 'true'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.10)

"github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)